### PR TITLE
update api key

### PR DESCRIPTION
**Pull request by inigomedina**
We need to add the new API key for Google Maps examples. This is the first update to see if it works properly. If it works, I'm going to update the rest of the examples.

### CODE REVIEW

**Comment by VictorVelarde** [comment]
Hi @inigomedina , thx for pushing this!

After reading the original issue (https://github.com/CartoDB/developers/issues/514), I think we are going to tackle this from RT, so I'm pinging here @rjimenezda 

The main steps are probably pretty clear:
**carto.js**
[ ] Take this PR and test it locally
[ ] Ensure the API_KEY (and proper GMaps version!) is used in all carto.js examples. Currently GMaps v3.35 is the choice (with v3.36 coming soon). See https://developers.google.com/maps/documentation/javascript/versions for reference.

**developers**
[ ] New PR, to deploy this provisional branch to staging and review the carto.js examples there
[ ] If everything is fine, merge PR in carto.js > new version release (patch?)
[ ] And finally merge developers PR to deploy the changes to fix this problem

If any situation arises with this, we can make the proper updates here, or talk together.

**Comment by inigomedina** [comment]
Thanks a lot, @VictorVelarde. The plan sounds perfect. 🔝

**Comment by rjimenezda** [comment]
Hi @inigomedina 

I'm trying this locally, and it seems like localhost is not allowed as a host. Could you please try to authorize it?

image

This doesn't happen with the old keys, I get a quota error.

If you still have access to the old key, please make sure that all the allowed referers are the same. I can take care of updating everything.

@VictorVelarde So I should upgrade the version as well? It's 3.32 right now

**Comment by inigomedina** [comment]
> I'm trying this locally, and it seems like localhost is not allowed as a host. Could you please try to authorize it?

@jvillarf was the person in charge of that part [on this issue](https://github.com/CartoDB/developers/issues/514). He precisely talked about this, because he restricted referrers to `http|https://carto.com/developers/*`.

cc @jvillarf

**Comment by jvillarf** [comment]
@inigomedina @ramiroaznar i've just disabled the **referrer** restriction, so you can test it locally. Would you please go ahead? (GCloud dashboar says the change can take up to 5 minutes, so be patient).

Once you have finished and we are ready for Production, please ping me again, so i can set the referrer restriction again.

Thanks.

**Comment by ramiroaznar** [comment]
How can I test this?

**Comment by jvillarf** [comment]
Sorry @ramiroaznar i wanted to ping @rjimenezda instead of you and autocomplete played with me :crying_cat_face:

**Comment by rjimenezda** [comment]
Thanks @jvillarf, I'll go ahead and change it for all. Will ping you as soon as we can block localhost again

File: examples/public/layers/change-feature-columns-gmaps.html [hunk 1]
```diff
@@ -104,12 +104,12 @@ <h1>Change the feature columns</h1>
 
     function setMoreData() {
       layer.setFeatureClickColumns(['name', 'pop_max', 'pop_min']);
-      document.getElementById('content').innerHTML = '';
+      document.getElementById('info').innerHTML = '';
```
Comment (VictorVelarde) [inline examples/public/layers/change-feature-columns-gmaps.html]: Was this broken because of a wrong id?

Comment (rjimenezda) [reply examples/public/layers/change-feature-columns-gmaps.html]: Yeah, but nothing serious, that's only used to "clear" a text. I noticed when checking all examples.

Comment (VictorVelarde) [reply examples/public/layers/change-feature-columns-gmaps.html]: Good boy scout 😄 !!

**Comment by rjimenezda** [comment]
https://carto.com/developers/carto-js/examples/#example-add-a-layer-gmaps

**Comment by inigomedina** [comment]
Nice! 👏

**Comment by ramiroaznar** [comment]
:boom:

**Comment by jvillarf** [comment]
d

Metrics show `200 OK` already.

**Pull request merged**